data/questions_gen_postgres.csv

```diff
@@ -112,7 +112,7 @@ What are the top 5 cities with the highest population? Give both city names and
 "What are the highest mountains in meters, ordered from highest to lowest altitude?","SELECT mountain.mountain_name, mountain.mountain_altitude FROM mountain ORDER BY mountain.mountain_altitude DESC NULLS LAST;",geography,order_by,
 What is the ratio of the population of the United States to the population of California?,"SELECT CAST(SUM(NULLIF(state.population, 0)) FILTER (WHERE LOWER(state.country_name) LIKE '%united states%') AS FLOAT) / CAST(SUM(NULLIF(state.population, 0)) FILTER (WHERE LOWER(state.state_name) LIKE '%california%') AS FLOAT) AS population_ratio FROM state;",geography,ratio,
 What is the ratio of the length of the Mississippi River to the length of the Rhine River?,"SELECT CAST((SELECT LENGTH FROM river WHERE LOWER(river_name) LIKE '%mississippi%') AS FLOAT) / NULLIF((SELECT LENGTH FROM river WHERE LOWER(river_name) LIKE '%rhine%'), 0) AS ratio ;",geography,ratio,
-What is the ratio of the altitude of Mount Everest to the altitude of Dhaulagiri?,"SELECT (CAST(everest.mountain_altitude AS FLOAT) / NULLIF(dhaulagiri.mountain_altitude, 0)) AS altitude_ratio FROM (SELECT mountain_altitude FROM mountain WHERE LOWER(mountain_name) LIKE '%mount everest%') AS everest, (SELECT mountain_altitude FROM mountain WHERE LOWER(mountain_name) LIKE '%dhaulagiri%') AS dhaulagiri;",geography,ratio,
+"What is the ratio of the altitude of 'Mount Everest' to the altitude of 'Dhaulagiri'? Match strings exactly","SELECT (CAST(everest.mountain_altitude AS FLOAT) / NULLIF(dhaulagiri.mountain_altitude, 0)) AS altitude_ratio FROM (SELECT mountain_altitude FROM mountain WHERE mountain_name = 'Mount Everest') AS everest, (SELECT mountain_altitude FROM mountain WHERE mountain_name = 'Dhaulagiri') AS dhaulagiri;",geography,ratio,

```

Comment:
There is an implicit string matching rule here, so I just made it explicit. This also lets the model know to match on the quoted strings as opposed to just `'Everest'`, since our column description for `mountain_name` doesn't mention whether the 'mount' word would be included or not.